codecov.yml

```
@@ -0,0 +1,7 @@
+comment:
+  layout: "header, reach, diff, files"
+  behavior: default
+  require_changes: false
+  require_base: false
+  require_head: true
+  hide_project_coverage: false
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)